src/main/scala/outwatch/dom/helpers/EmitterBuilder.scala

```diff
@@ -33,7 +33,7 @@ final case class EmitterBuilder[E, O] private (
 
 object EmitterBuilder {
   def apply[E <: Event](eventType: String): EmitterBuilder[E, E] = new EmitterBuilder[E, E](
-    eventType, identity, (e, o) => o.next(e.asInstanceOf[E])
+    eventType, identity, (e, o) => o.asInstanceOf[Observer[Event]].next(e)
```

Comment:
Why are you casting the `Observer` instead of the `Event`?

Reply:
I was actually casting the event at one point and I was getting a weird random test failure, then I reverted to casting the observer. The failure was probably unrelated though and I believe the two ways should be equivalent (because Observer is contravariant).